File: acr2016/go/main.go
package main

import (
	"fmt"

	"github.com/cznic/mathutil"
)

func main() {
	var paths int
	for i := uint32(0); i < uint32(1<<20); i++ {
		bitCount := mathutil.PopCountUint32(i)
		if bitCount == 10 {
			paths++
		}
	}

	fmt.Println(paths)
}

File: acr2016/main.c
#include <stdio.h>

int main() {
  int paths = 0;
  for(int i = 0; i < (1<<20); i++) {
    int bitCount = __builtin_popcount(i);
    if(bitCount == 10) {
      paths += bitCount;
    }
  }

  printf("%d\n", paths);
  return 0;
}
